subworkflows/nf-core/bam_variant_calling_freebayes/main.nf

```diff
@@ -0,0 +1,43 @@
+include { SAMTOOLS_FAIDX } from '../../../modules/nf-core/samtools/faidx/main'
+include { FREEBAYES      } from '../../../modules/nf-core/freebayes/main'
+include { BCFTOOLS_INDEX } from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_SORT  } from '../../../modules/nf-core/bcftools/sort/main'
+
+workflow BAM_VARIANT_CALLING_FREEBAYES {
+
+    take:
+    ch_input        // channel: [ val(meta), bam/cram, bai/crai, bam/cram, bai/crai, bed ]
```

Comment:
What is bam/cram, bai/crai there twice?

Reply:
For somatic variant calling, it's in the tests, we discussed about that already, we merged it months ago

Reply:
Maybe label accordingly :)

Reply:
Added comments in `meta.yml` to clarify.